WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingAddPackageViewModel.swift

```diff
@@ -212,7 +212,16 @@ final class WooShippingAddPackageViewModel: ObservableObject {
             self.selectedSavedPackageId = nil
         }
 
-        return nil
+        let deleteAction = WooShippingAction.deletePackage(siteID: siteID, packageID: packageToRemove.id) { result in
+            switch result {
+            case .success(let response):
+                self.transformSavedPackages(response)
+            case .failure:
+                break
+            }
+        }
+
+        ServiceLocator.stores.dispatch(deleteAction)
```

Comment:
How about using the view model's `stores` property (instead of the `ServiceLocator`) for ease of unit testing?